pkg/types/room.go

```diff
@@ -99,7 +100,7 @@ func (r *Room) syncPeerLists() {
 	for _, peer := range r.Peers {
 		r.SendMessageToAllPeers(MessageContent{
 			Type: MessageTypeCmd,
-			Data: []byte("join "+peer.RIdentity.Fingerprint()),
+			Data: []byte(string(RoomCommandJoin) + peer.RIdentity.Fingerprint()),
```

Comment:
I think we need a space here

Reply:
gofmt says its fine ;)

Reply:
`Data: []byte(string(RoomCommandJoin) + " " + peer.RIdentity.Fingerprint()),`
We need the space as delimiter
Or add a space to the RoomCommands